falcon/__init__.py

```diff
@@ -23,8 +23,6 @@
     app = falcon.App()
 """
 
-import logging as _logging
```

Comment:
I don't understand this change. It's not related to types, so could you revert?

Reply:
The reason for this change: the logger is used in asgi.app like that:
https://github.com/falconry/falcon/blob/61c1047db2ef02556da9c28cc42c813b52db3aa9/falcon/asgi/app.py#L1188
but there is no 'import falcon' anythere is this file, so _logger in fact can't be accessed. I think it's the bug actually.
If I add 'import falcon' at top of 'asgi.app', it could lead to circular import if asgi is ever imported in main falcon 
package (now it seems to be not). Should I change it to just `from falcon import _logger` to the top of `asgi.app` ?

Reply:
I think it's liked by by python and type checker because there is this import in that file:https://github.com/falconry/falcon/blob/61c1047db2ef02556da9c28cc42c813b52db3aa9/falcon/asgi/app.py#L51

Reply:
but let's import it explicitely

Reply:
Moved the _logger back to the main __init__.py.
It's explicitly imported in asgi.app now